code/modules/projectiles/guns/ds13/line_cutter.dm

```diff
@@ -14,7 +14,6 @@
 	icon_state = "linecutter"
 	item_state = "linecutter"
 
-	ammo_type = /obj/item/ammo_casing/linerack
```

Comment:
This seems to be removed. Why is that?

Reply:
duplicate var:
![image](https://user-images.githubusercontent.com/49106866/108898719-d8e2ce00-7617-11eb-9cad-1bca3c09c379.png)

Reply:
![image](https://user-images.githubusercontent.com/49106866/108898777-e8faad80-7617-11eb-921f-062f6f96d3cc.png)

Reply:
even so, I had forgotten this change, I did it when I was trying to apply this fix in a different way.